File: src/main/java/com/xxy/Student/Student.java
package com.xxy.Student;

public class Student {
    String id;
    String name;
    int english;
    int math;

    public Student(String name, int english, int math) {
        this.name = name;
        this.english = english;
        this.math = math;
    }

    public int highest(){
        return (english > math) ? english : math;
    }

    public void print(){
        System.out.print(name+"\t"+english+"\t"+math+"\t"+getAverage());
        if (getAverage() >= 60){
            System.out.println("\tPASS");
        }
        else{
            System.out.println("\tFAILED");
        }
    }

    public int getAverage(){
        return (english+math)/2;
    }
}

File: src/main/kotlin/com/xxy/Student/Student.kt
package com.xxy.Student

import java.util.*

fun main(args:Array<String>) {
//
    val stu = Student("XXY", 90, 100)
    stu.print()
    println("High score:${stu.highest()}")
}

class Student(var name:String, var english:Int, var math:Int){
    fun print(){
        print(name+"\t"+english+"\t"+math+"\t"+getAverage()+if(getAverage()>=60)"PASS" else {
            "FAILED"
        })
    }

    fun getAverage():Int{
        return (english+math)/2
    }

    fun highest():Int{
        val max = if (english > math) {
            english
        } else {
            math
        }
        return max
    }
}


private fun Userinput() {
    val scanner = Scanner(System.`in`)
    println("Please input student's name:")
    val name = scanner.next()
    println("Please input student's englsih grade:")
    val english = scanner.nextInt()
    println("Please input student's math grade:")
    val math = scanner.nextInt()
    val stu = Student(name, english, math)
    stu.print()
}

